src/jit/gentree.cpp

```diff
@@ -18036,8 +18036,9 @@ bool GenTreeHWIntrinsic::OperIsMemoryLoadOrStore()
 {
 #ifdef _TARGET_XARCH_
     return OperIsMemoryLoad() || OperIsMemoryStore();
-#endif // _TARGET_XARCH_
+#else  // _TARGET_XARCH_
```

Comment:
The #else comment should be !_TARGET_XARCH_ (and endif as well).

Reply:
I think I'd rather remove the comments. For one line ifdefs like these they just to the noise.

Reply:
> For one line ifdefs like these they just to the noise.

I agree. The coding conventions also only indicate it is needed at the end of a long or nested `#if/#ifdef`: https://github.com/dotnet/coreclr/blob/c2baf04cd2c2211334949ba12df2e49fd9109728/Documentation/coding-guidelines/clr-jit-coding-conventions.md#75-commenting-ifdefs

Reply:
I also prefer to leave them off for small regions. As @mikedn points out, they just clutter things up and don't really add any clarity.

Reply:
OK, will remove.